.github/workflows/nightly.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     name: docs build
     uses: ./.github/workflows/_linux-build.yml
     with:
+      runner: "amz2023.linux.2xlarge"
```

Comment:
Here's an example of the old format, where you'd have to explicitly state the machine type.

I'll extend _linux-build and the other reusable workflows to use the new format in a separate PR